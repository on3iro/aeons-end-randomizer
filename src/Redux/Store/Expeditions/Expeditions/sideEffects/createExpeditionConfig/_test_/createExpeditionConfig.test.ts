import { createExpeditionConfig } from '../index'

describe('createExpeditionConfig()', () => {
  it.todo('TODO')
})
