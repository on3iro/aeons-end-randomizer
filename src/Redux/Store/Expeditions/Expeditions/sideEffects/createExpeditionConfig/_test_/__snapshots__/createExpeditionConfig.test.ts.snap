// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createExpeditionConfig() should produce correct result 1`] = `
Object {
  "banished": Array [],
  "barracks": Object {
    "mageIds": Array [
      "Quilius",
      "Mist",
      "Gex",
      "Reeve",
    ],
    "supplyIds": Array [
      "VriswoodAmber",
      "BreachOre",
      "MentiteChunk",
      "FiendCatcher",
      "LinkConduit",
      "Conflagration",
      "VoidBond",
      "Resonate",
      "Equilibrium",
    ],
    "treasureIds": Array [
      "QuiliussErasure",
      "BramasBuriedLight",
      "IndirasLoneOpal",
      "GarusTorch",
      "SparrowsAid",
    ],
  },
  "battles": Array [
    Object {
      "expeditionId": "TEST_EXPEDITION_ID",
      "id": "TEST_EXPEDITION_ID",
      "nemesisTier": Object {
        "isNewTier": true,
        "tier": 2,
      },
      "status": "unlocked",
      "treasure": Object {
        "hasTreasure": true,
        "level": 2,
      },
      "tries": 0,
    },
    Object {
      "expeditionId": "TEST_EXPEDITION_ID",
      "id": "TEST_EXPEDITION_ID",
      "nemesisTier": Object {
        "isNewTier": true,
        "tier": 3,
      },
      "status": "locked",
      "treasure": Object {
        "hasTreasure": true,
        "level": 3,
      },
      "tries": 0,
    },
    Object {
      "expeditionId": "TEST_EXPEDITION_ID",
      "id": "TEST_EXPEDITION_ID",
      "nemesisTier": Object {
        "isNewTier": true,
        "tier": 4,
      },
      "status": "locked",
      "treasure": Object {
        "hasTreasure": false,
      },
      "tries": 0,
    },
  ],
  "bigPocketVariant": true,
  "finished": false,
  "id": "TEST_EXPEDITION_ID",
  "name": "Test expedition config",
  "score": 0,
  "seed": Object {
    "nemesisState": true,
    "seed": "TEST",
    "supplyState": Object {
      "S": Array [
        219,
        164,
        228,
        57,
        11,
        149,
        148,
        134,
        55,
        4,
        15,
        223,
        45,
        213,
        242,
        109,
        72,
        185,
        240,
        14,
        118,
        34,
        106,
        88,
        211,
        23,
        135,
        49,
        188,
        65,
        174,
        54,
        16,
        39,
        151,
        129,
        165,
        162,
        75,
        52,
        189,
        122,
        59,
        103,
        58,
        202,
        144,
        197,
        50,
        113,
        155,
        210,
        225,
        71,
        100,
        154,
        115,
        56,
        224,
        168,
        91,
        40,
        85,
        0,
        160,
        247,
        73,
        24,
        207,
        31,
        150,
        83,
        101,
        138,
        37,
        98,
        236,
        6,
        3,
        241,
        35,
        178,
        93,
        177,
        141,
        26,
        184,
        201,
        69,
        112,
        114,
        8,
        147,
        235,
        29,
        89,
        119,
        173,
        204,
        217,
        51,
        116,
        120,
        127,
        47,
        28,
        176,
        231,
        1,
        159,
        146,
        172,
        208,
        181,
        221,
        7,
        77,
        244,
        251,
        2,
        248,
        198,
        5,
        186,
        167,
        179,
        206,
        137,
        99,
        230,
        169,
        36,
        156,
        87,
        22,
        33,
        193,
        200,
        214,
        27,
        250,
        190,
        253,
        249,
        62,
        131,
        157,
        199,
        68,
        30,
        170,
        143,
        238,
        41,
        60,
        227,
        161,
        145,
        218,
        175,
        209,
        123,
        171,
        86,
        142,
        21,
        192,
        95,
        25,
        61,
        132,
        158,
        67,
        63,
        105,
        136,
        92,
        139,
        102,
        97,
        53,
        18,
        78,
        43,
        246,
        195,
        234,
        38,
        128,
        66,
        81,
        243,
        183,
        108,
        74,
        226,
        124,
        216,
        19,
        166,
        180,
        79,
        17,
        42,
        182,
        187,
        222,
        126,
        111,
        203,
        82,
        13,
        32,
        46,
        133,
        254,
        48,
        232,
        215,
        12,
        191,
        252,
        20,
        90,
        94,
        110,
        80,
        130,
        70,
        196,
        44,
        140,
        121,
        205,
        239,
        212,
        125,
        229,
        76,
        84,
        194,
        237,
        233,
        117,
        153,
        107,
        255,
        10,
        220,
        245,
        96,
        152,
        64,
        104,
        9,
        163,
      ],
      "i": 126,
      "j": 243,
    },
  },
  "settingsSnapshot": Object {
    "availableCardIds": Array [
      "DiamondCluster",
      "ChaosArc",
      "Ignite",
      "EssenceTheft",
      "SearingRuby",
      "FeralLightning",
      "PlanarInsight",
      "SpectralEcho",
      "BurningOpal",
      "ConsumingVoid",
      "UnstablePrism",
      "CloudedSapphire",
      "MagesTalisman",
      "FlexingDagger",
      "LavaTendril",
      "BottledVortex",
      "ArcaneNexus",
      "DarkFire",
      "PhoenixFlame",
      "Jade",
      "AmplifyVision",
      "VriswoodAmber",
      "BlastingStaff",
      "SiftersPearl",
      "WildfireWhip",
      "FocusingOrb",
      "OblivionSwell",
      "TripliteCore",
      "NerveJab",
      "DoubleTap",
      "ObliviumResin",
      "Summonite",
      "ManifoldContainer",
      "DrainingTouch",
      "JoltingCrust",
      "MentiteChunk",
      "SoulCords",
      "VoidiumSpike",
      "PrecisionShot",
      "MutedLacosite",
      "BladeOfWisdom",
      "StunningForce",
      "FlashOfIntellect",
      "ScholarsOpus",
      "BottledStar",
      "BurningCurrent",
      "PsychicEruption",
      "LivingGauntlet",
      "ShiningTetrite",
      "VoidBond",
      "Combustion",
      "VimDynamo",
      "DisintegratingScythe",
      "MonstrousInferno",
      "DevouringShadow",
      "BanishingTopaz",
      "Transmogrifier",
      "JeweledBrain",
      "BlastSphere",
      "WoundMender",
      "LurkingSionite",
      "UnrefinedBlaststone",
      "Exogranite",
      "RecurringJasper",
      "EnergizedRubidium",
      "ResonantPearl",
      "CagedFire",
      "GalvanizedBauble",
      "AetherConduit",
      "WellOfEnergy",
      "PainConduit",
      "LinkConduit",
      "MarbleGalaxy",
      "DelugeOfPower",
      "ReverberatingShock",
      "FatalHarmony",
      "BouncingBoom",
      "SpiritLift",
      "Tethered Darts",
      "ForceAmplifier",
      "ImbuedSmash",
      "PatternStrike",
      "MoltenHammer",
      "Radiance",
      "Blaze",
      "ScryingBolt",
      "SagesBrand",
      "LeechingAgate",
      "TemporalHelix",
      "Char",
      "AlienElement",
      "Scorch",
      "Pyromancy",
      "FeedbackAura",
      "Catalyst",
      "PainStone",
      "AstralCube",
      "RiddleSphere",
      "NetherConduit",
      "HauntedBerylite",
      "Olivinite",
      "CacheGlass",
      "ReflectiveConduit",
      "WillWeaver",
      "BreachSeeker",
      "CrescendoRay",
      "StormVapors",
      "MantraOfStrength",
      "EmbodyFlame",
      "SnapRitual",
      "ShiningFluorite",
      "PolyphaseTurbine",
      "BreachExtractor",
      "FocusingConduit",
      "DualFlash",
      "DustCaller",
      "SymbioticSynapse",
      "FossilizedScarab",
      "ThermalDart",
      "Resonate",
      "Conflagration",
      "DimensionalKey",
      "InnerFire",
      "Fulminate",
      "EternityCharm",
      "ErraticIngot",
      "ReduceToAsh",
      "VortexGauntlet",
      "Carbonize",
      "Equilibrium",
      "VolcanicGlass",
      "ConvectionField",
      "JaggedLightning",
      "BloodstoneJewel",
      "Kindle",
      "ThoughtformFamiliar",
      "Crystallize",
      "FieryTorrent",
      "CelestialSpire",
      "ConjureTheLost",
      "NovaForge",
      "DreadDiamond",
      "CairnCompass",
      "ScoriaSlag",
      "BreachOre",
      "FiendCatcher",
      "PyrotechnicSurge",
      "Aurora",
      "ConclaveScroll",
      "PrimordialFetish",
      "FrozenMagmite",
      "MagesTotem",
      "DrownInFlames",
      "FleetingVision",
      "SplinterMissile",
      "EchoStone",
      "ThievingSpirit",
      "FracturedQuartz",
      "RockLauncher",
      "AbacusOfIgnition",
      "IlluminatingFlame",
      "ChronalArc",
    ],
    "availableMageIds": Array [
      "Adelheim",
      "Brama",
      "Jian",
      "Kadir",
      "Lash",
      "Mist",
      "Phaedraxa",
      "Xaxos",
      "Nym",
      "Reeve",
      "Zhana",
      "Inco",
      "Razra",
      "Sahala",
      "Gygar",
      "Soskel",
      "Taqren",
      "Talix",
      "Rhia",
      "Claudia",
      "Lost",
      "Malastar",
      "Indira",
      "Remnant",
      "Nook",
      "Mazra",
      "Qu",
      "Sparrow",
      "XaxosTV",
      "Dezmodia",
      "Garu",
      "Gex",
      "Mazahaedron",
      "MistWE",
      "Quilius",
      "Ulgimor",
      "YanMagda",
    ],
    "availableNemesisIds": Array [
      "CarapaceQueen",
      "CrookedMask",
      "PrinceOfGluttons",
      "Rageborne",
      "HordeCrone",
      "Necroswarm",
      "Maggoth",
      "Arachnos",
      "AgelessWalker",
      "Fenrix",
      "BlightLord",
      "WaywardOne",
      "ThriceDeadProphet",
      "Wraithmonger",
      "TheWailing",
      "TheWanderer",
      "KnightOfShackles",
      "MaidenOfThorns",
      "GateWitch",
      "HollowCrown",
      "MagusOfCloaks",
      "UmbraTitan",
    ],
    "availableTreasureIds": Array [
      "DezmodiasOblivionShard",
      "UlgimorsCoalFragment",
      "IndirasLoneOpal",
      "MistsAmethystParagon",
      "MalastarsImmolate",
      "YanMagdasIlluminate",
      "EyeOfTheMaelstrom",
      "WellOfDespair",
      "CronesAmulet",
      "EdibleFungusChunks",
      "UmbralHornOfWar",
      "TheBrokenMask",
      "AlchemistsAlembic",
      "SoothsayersPouch",
      "SuppressingPin",
      "Quicksilver",
      "BandOfRetrieval",
      "PurifiedBangle",
      "KadirsEmeraldShard",
      "GexsShatteredGeode",
      "JiansMoonstoneShard",
      "LashsQuartzShard",
      "XaxosFlare",
      "AdelheimsAmethystShard",
      "BramasBuriedLight",
      "GarusTorch",
      "XaxosPyre",
      "MistsGarnetShard",
      "ShimmeringCloakOfTheMagus",
      "ImbuedShackles",
      "GluttonsTooth",
      "CoreOfRage",
      "CleansingAmulet",
      "CarapaceFragement",
      "ThornedWhip",
      "BonesOfDeathmind",
      "AcceleratingGauntlets",
      "EssenceExtractor",
      "BroochOfAttunment",
      "BladedCrystal",
      "PrismOfDestruction",
      "ForgottenTrinket",
      "PrecisionMagnifier",
      "ShroudOfObfucation",
      "VerdantStaff",
      "FlowingMantle",
      "MazahaedronsWorldheartShard",
      "PhaedraxasTourmalineShard",
      "FracturedShell",
      "Extra-DimensionalLens",
      "SeersBracer",
      "RemnantsVoidShard",
      "QuiliussErasure",
      "ZhanasFleetingEmber",
      "BlightedRootClump",
      "PetrifiedWitchFinger",
      "WaywardScraps",
      "TrueSightMonocle",
      "EntwinedAetherStrands",
      "SiphoningBlade",
      "GemEncrustedAnklet",
      "ReevesRavenShard",
      "NymsVision",
      "SparrowsAid",
      "WraithsEssence",
      "VolatileClasp",
      "BreachStabilizer",
      "FangedChoker",
      "EndlessBandolier",
    ],
    "availableUpgradedBasicNemesisCardIds": Array [
      "HissingAcid",
      "Wreck",
      "BaneCommander",
      "SkyTremor",
      "EngulfingMadness",
      "Separate",
      "NeedleDoom",
      "Marked",
      "Eradicate",
      "FlashOfDecay",
      "Terrify",
      "RayOfDesolation",
      "ThunderousTempest",
      "Sacrifice",
      "Burialskulk",
      "Rupture",
      "FurrowOfDestruction",
      "Mangle",
      "HowlingSlayer",
    ],
    "supplySetup": Object {
      "active": false,
      "id": "market1",
      "name": "Market Setup 1",
      "tiles": Array [
        Object {
          "operation": "<",
          "threshold": 4,
          "type": "Gem",
        },
        Object {
          "operation": "=",
          "threshold": 4,
          "type": "Gem",
        },
        Object {
          "operation": "ANY",
          "type": "Gem",
        },
        Object {
          "operation": "ANY",
          "type": "Relic",
        },
        Object {
          "operation": "ANY",
          "type": "Relic",
        },
        Object {
          "operation": "<",
          "threshold": 5,
          "type": "Spell",
        },
        Object {
          "operation": "<",
          "threshold": 5,
          "type": "Spell",
        },
        Object {
          "operation": ">",
          "threshold": 5,
          "type": "Spell",
        },
        Object {
          "operation": ">",
          "threshold": 5,
          "type": "Spell",
        },
      ],
      "type": "official",
    },
    "usedExpansions": Array [
      "AE",
      "BS",
      "Depths",
      "IW",
      "NA",
      "Nameless",
      "OD",
      "SD",
      "TA",
      "TV",
      "WE",
      "promos",
    ],
  },
  "upgradedBasicNemesisCards": Array [],
  "variantId": "SHORT",
}
`;
